commands.py

```diff
@@ -1371,53 +1373,6 @@ def wordoftheday(message_dict):
 	send_message(f"{tag}The Spanish Word of the Day is \"{spa}\", which means \"{eng}\"")
 	log(f"Sent word of the Day to {user}: {spa} means {eng}")
 
-@is_command("Show the price of bitcoin")
-def btc(message_dict):

```

Comment:
Oh, I think we should also keep the btc, eth, and doge commands as Kaywee uses them often. And I think having a special command for the 2-3 coins of most interest to the chat is a good idea, bc it reduces the length of the command from e.g. `!crypto eth` to just `!eth`. Then for any other less-common coins chatters can still use `!crypto <coin>`, or for the three common ones they can still use the `!crypto` shortcut.